GoogleMapsApi/Entities/TimeZone/Response/TimeZoneResponse.cs

```diff
@@ -30,7 +30,7 @@ internal string StatusStr
         /// DstOffset: the offset for daylight-savings time in seconds. This will be zero if the time zone is not in Daylight Savings Time during the specified timestamp.
         /// </summary>
         [DataMember(Name = "dstOffset")]
-        public double OffSet { get; set; }
+        public double DstOffSet { get; set; }
```

Comment:
This's a breaking change. 

If you think that this change is necessary, what I suggest to do is add a new prop `DstOffSet as` you did **but** keep the old one (delegating to the new one) and mark it as -
[Obsolete("Use DstOffSet instead")]

This way you won't break compilation for existing users and give them some time to migrate.
After a while, we'll be able to delete the [Obsolete] method.